WeeklyTask/WeeklyTaskService.cs

```diff
@@ -6,103 +6,160 @@
 
 namespace WeeklyTask
 {
+    delegate void GetMessage(string message);
+    delegate void WriteOutput(string text);
+    delegate string ReadInput();
+    
     internal class WeeklyTaskService
     {
+        GetMessage _mes= GetMessageAboutUpdateTask;
+        WriteOutput _writeOutput;
+        ReadInput _readInput=()=>Console.ReadLine();
         private  int _counter;
         private readonly WeeklyTask[] _tasks = new WeeklyTask[10];
+        public void RegisterExchanger(WriteOutput writeOutput)
```

Comment:
call RegisterExchanger from `Program.cs` and pass there `Console.WriteLine` reference